Sources/PT.PM.AntlrUtils/AntlrConverter.cs

```diff
@@ -263,6 +263,12 @@ protected Ust DefaultResult
             return CreateBinaryOperatorExpression(left, operatorTerminal.GetText(), operatorTerminal.GetTextSpan(),  right);
         }
 
+        protected virtual BinaryOperator GetBinaryOperatorFromCombined(string combinedOperatorText)
+        {
+            var operatorText = combinedOperatorText.TakeWhile(x => !x.Equals('=')).ToArray();
+            return CreateBinaryOperator(new string(operatorText));
```

Comment:
I think the following code is more clear and faster.

```csharp
combinedOperatorText.Remove(combinedOperatorText.IndexOf('='))
```

Moreover, I think the single `=` character will be always at the end. Thus, we can use more simplified code:

```csharp
combinedOperatorText.Remove(combinedOperatorText.Length - 1);
```